tests/test_taxi.py

```diff
@@ -69,7 +67,7 @@ def test_car_is_register_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
Consider using `self.assertTrue(file_exists)` instead of `assert file_exists` for consistency with Django's `TestCase` assertions. This provides better error messages if the test fails.